.github/workflows/on-push.yaml

```diff
@@ -86,11 +86,11 @@ jobs:
           aws eks update-kubeconfig --alias hidden --name platform-dev2 --region us-east-1;
           kubectl -n qctrl-platform set image deployment/python-open-controls-docs python-open-controls-docs=qctrl/python-open-controls-docs:$COMMIT && kubectl -n qctrl-platform rollout status deployment/python-open-controls-docs;
         '
-      - name: Update Sphinx deployment in Kubernetes (Production)
-        run: |
-          ./ci vault awsKey -t $(cat .token) -r deploy-eks-prod -o ./.aws-credentials
-          ./ci docker run qctrl/ci-images:google-cloud-sdk-ci sh -- -c '
-            . ./.aws-credentials;
-            aws eks update-kubeconfig --alias hidden --name platform-prod --region us-west-2;
+    - name: Update Sphinx deployment in Kubernetes (Production)
```

Comment:
For my education, is this for some test?

Reply:
This is deploying to production without a release ;)

Reply:
o_o